examples/web/RunAnywhereAI/src/styles/components.css

```
@@ -0,0 +1,1756 @@
+.setting-value { font-size: var(--font-size-base); color: var(--text-secondary); }
+
+/** Stacked variant for form field rows (API key, URL, etc.) */
+.setting-row--stacked {
+  flex-direction: column;
+  align-items: stretch;
+  gap: var(--space-sm-md);
+}
+
+/* Slider */
+input[type="range"] {
+  -webkit-appearance: none;
```

Stylelint (17.2.0)

[error] Unexpected vendor-prefixed property "-webkit-appearance" (property-no-vendor-prefix)

(property-no-vendor-prefix)